backend/lib/services/members/MemberManager.js

```diff
@@ -169,6 +184,21 @@ class MemberManager {
     this.subjectList.delete(item.id)
   }
 
+  async deleteServiceAccountSecret (item) {
+    const { namespace } = Member.parseUsername(item.id)
+    if (namespace !== this.namespace) {
+      return
```

Comment:
This should be an error I think?!

Reply:
Yes, we can throw an error here. Also 422 in this case?